proto/validator/accounts/v2/web_api.proto

```diff
@@ -18,6 +18,11 @@ service Wallet {
             body: "*"
         };
     }
+    rpc DefaultWalletPath(google.protobuf.Empty) returns (DefaultWalletResponse) {
+        option (google.api.http) = {
+            get: "/v2/validator/wallet/default",
```

Comment:
```suggestion
            get: "/v2/validator/wallet/config/default",
```
Recommend: add config to the path for clarity. 
I was thinking that `/v2/validator/wallet` would return a default wallet, so the distinction wasn't very clear between those two API routes.